spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -120,14 +120,14 @@ def process(options)
 
       context "when process is interupted twice" do
         before do
-          subject.demultiplexer.stub(:stop) # do nothing just ignore the interuption
-          subject.stub(:exit){ throw :got_forcefully_stopped }
+          allow(subject.demultiplexer).to receive(:stop) # do nothing just ignore the interuption
+          allow(subject).to receive(:exit) { throw :got_forcefully_stopped }
 
           stub_const("TwiceInterupter", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              2.times { Process.kill("INT", 0) }
-            end
-          })
+                                        def process(options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `process(*)` if you want the method to accept any arguments but don't care about them.